src/models/git-hub-server-app.ts

```diff
@@ -64,6 +64,20 @@ export class GitHubServerApp extends Model {
 
 		return gitHubServerApp;
 	}
+
+	/**
+	 * Get GitHubServerApp
+	 *
+	 * @param {{uuid: string}} uuid
+	 * @returns {GitHubServerApp}
+	 */
+	static async getGitHubServerAppForUuid(uuid: string): Promise<GitHubServerApp | null> {
```

Comment:
I'd probably name this `findForUuid` since you don't need the object context in the function name (already in the model class) and we are also more explicit as to which operation the function is doing (find)

Reply:
Updated method name to `findForUuid`